app/models/container_node.rb

```diff
@@ -85,4 +86,22 @@ def evaluate_alert(_alert_id, _event)
     # and these should automaticaly be translated to alerts.
     true
   end
+
+  supports :external_logging_support do
+    unless ext_management_system.respond_to?(:external_logging_route_name)
+      unsupported_reason_add(:external_logging_support, _('This provider type does not support Extrenal Logging'))
+    end
+  end
+
+  def external_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def external_logging_path
+    fqdn = (ipaddress || name || "")
+    hostname = fqdn.split('.').first
```

Comment:
@enoodle @portante if `fqdn` is an ipaddress... wouldn't `hostname` be the first octet?

Also shouldn't you protect against `hostname` possibly being `nil`?

If neither `ipaddress` or `name` are defined why do we continue with `""`? Wouldn't it generate a meaningless query?